interfaces/remoteOperatorUI/index.js

```diff
@@ -77,7 +77,18 @@ function startHTTPServer(localUIApp, port) {
     let mouseConnected = false;
     var callibrationFrames = 100;
 
-    const http = require('http').Server(localUIApp.app);
+    let http = null;
+    if (server8080.useHTTPS) {
+        const fs = require('fs');
+        const path = require('path');
+        let options = {
+            key: fs.readFileSync('key.pem'),
```

Comment:
It would be more straightforwards if these paths pointed to the server's key.pem and cert.pem instead of requiring manually copying the files

Reply:
You needed to copy those files?